src/app/error.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import React, { useState } from "react";
+import Button from "@mui/material/Button";
+import {
+    ErrorCenterer,
+    ErrorPanel,
+    ErrorLargeText,
+    ErrorSecondaryText,
+    ErrorRetryText,
+} from "@/app/errorPageStyling";
+
+interface ErrorProps {
+    error: Error;
+    reset: () => void;
+}
+
+const Error: React.FC<ErrorProps> = ({ error, reset }) => {
+    const [ErrorMessage, SetErrorMessage] = useState("");
+
+    const retry = (): void => {
+        SetErrorMessage("Retrying...");
+        setTimeout(() => {
+            reset();
+        }, 1000);
+    };
+    return (
+        <ErrorCenterer>
+            <ErrorPanel elevation={5}>
```

Comment:
The error modal is overflowing in mobile mode - maybe gotta set something like `width=max(...px, ...vw)` stuff

Reply:
Good point, have changed.